backend/PictoPy.spec

```
@@ -0,0 +1,60 @@
+# -*- mode: python ; coding: utf-8 -*-
+import os
+
+a = Analysis(
+    ['main.py'],
+    pathex=[],
+    binaries=[],
+    datas=[('app', 'app')],
```

RuboCop (1.86.1)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)